scripts/microbe_stage/microbe_operations.as

```diff
@@ -303,7 +303,6 @@ void respawnPlayer(CellStageWorld@ world)
     {
         playerSpecies.population = 0;
     }
-
```

Comment:
Completely unnecessary to remove one empty line from a file that you haven't otherwise edited.